src/blocks/blocks/popup/inspector.js

```diff
@@ -20,6 +20,13 @@ import {
 
 import { Fragment } from '@wordpress/element';
 
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+
+const { attributes: defaultAttributes } = metadata;
+
```

Comment:
Any reason to import metadata here? Don't see it being anywhere..

Reply:
No, just forgot it there.